app/processors/base_processor.py

```diff
@@ -36,31 +36,35 @@ def __init__(self, path, endpoint):
             headers = headers_for_development(headers)
             logger.debug('development headers: {}'.format(json.dumps(headers)))
 
-        # Respond to requester
-        response = requests.request(
-            request.method,
-            self.endpoint,
-            headers=headers,
-            params=request.args,
-            data=(await request.data),
-            stream=True,
-            timeout=300
-        )
-
-        logger.debug('Response: {}'.format(response.status_code))
-        logger.debug('Response headers: {}'.format(response.headers))
-
-        return Response(
-            response=self.generate_response_data(response),
-            headers=self.create_response_headers(response),
-            status=response.status_code,
-        )
-
-    async def generate_response_data(self, response):
-        for c in response.iter_lines():
-            # logger.debug(c)
-            yield c + "\r\n".encode()
-        yield "\r\n".encode()
+
+
+        async with aiohttp.ClientSession() as session:
+            async with session.request(
+                        request.method,
+                        self.endpoint,
+                        headers=headers,
+                        params=request.args,
+                        data=(await request.data),
+                    ) as response:
+
+                response_data = await response.read()
+
+                logger.debug('Response: {}'.format(response.status))
+                logger.debug('Response headers: {}'.format(response.headers))
+
+                return Response(
+                    # response=response_generator(response),
+                    response=response_data,
+                    headers=self.create_response_headers(response),
+                    status=response.status
+                )
+
+    # TODO: This does not work yet.
+    # TODO: Enable streaming responses at a later point.
```

Comment:
maybe you want to use https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.StreamResponse.write